CHANGELOG.md

```diff
@@ -40,7 +40,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where the Harvard RTF exporter did not use the new authors formatter and therefore did not export "organization" authors correctly. [4508](https://github.com/JabRef/jabref/issues/4508)
 - We fixed an issue where the field `urldate` was not exported to the corresponding fields `YearAccessed`, `MonthAccessed`, `DayAccessed` in MS Office XML [#7354](https://github.com/JabRef/jabref/issues/7354)
 - We fixed an issue where the password for a shared SQL database was only remembered if it was the same as the username [#6869](https://github.com/JabRef/jabref/issues/6869)
-- We fixed an issue where the file path is invisible in dark theme. [#7382](https://github.com/JabRef/jabref/issues/7382)
+- We fixed an issue with endless exception dialogues appearing if the file path includes an illegal char. [#7381](https://github.com/JabRef/jabref/issues/7381)
```

Comment:
Please do not delete existing changelog entries. The "trick" is to add the change in a random position in the list (and not to the end)

I readded it in this suggestion:

```suggestion
- We fixed an issue where the file path is invisible in dark theme. [#7382](https://github.com/JabRef/jabref/issues/7382)
- We fixed an issue with endless exception dialogues appearing if the file path includes an illegal char. [#7381](https://github.com/JabRef/jabref/issues/7381)
```